plugins/catalog-backend/src/next/Stitcher.ts

```diff
@@ -59,9 +60,22 @@ export class Stitcher {
 
   async stitch(entityRefs: Set<string>) {
     for (const entityRef of entityRefs) {
-      await this.transaction(async txOpaque => {
-        const tx = txOpaque as Knex.Transaction;
-
+      try {
+        const ticket = uuid();
+        const ticketRows = await this.database<DbFinalEntitiesRow>(
+          'final_entities',
+        )
+          .update('stitch_ticket', ticket)
+          .whereIn('entity_id', qb =>
```

Comment:
`db` 😅

Reply:
Nnäe. It stands for "query builder" (that's the type of the thing)